deploy/operator.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       serviceAccountName: smi-adapter-istio
       containers:
         - name: smi-adapter-istio
-          image: OPERATOR_IMAGE
+          image: quay.io/kinvolk/smi-adapter-istio:latest
```

Comment:
I would use sem ver or at least the git commit sha as the image tag.

Reply:
I have given `latest` tag there so that users can try the latest stuff without having to rebuild the image themselves. The problem with giving commit specific tag in this config is that we have to keep updating the tag everytime a new code is merged in `master`. Also this project is not doing releases yet which we could have used as tag to image and updated on each release, so I resorted to this method.

Is there a better way to give users latest stuff? Or adding commit tag to the image is not that very bad as I think? Or for now should we resort to use the `latest` tag and I create issue about doing biweekly releases and we use that as image tag?

Reply:
I would go with the short git commit.

Reply:
done